sklearn/metrics/tests/test_common.py

```diff
@@ -184,8 +184,8 @@
 # When you add a new metric or functionality, check if a general test
 # is already written.
 
-# Metric undefined with "binary" or "multiclass" input
-METRIC_UNDEFINED_MULTICLASS = [
+# Metric undefined with "binary"
+METRIC_UNDEFINED_BINARY = [
```

Comment:
The renaming is not true `"roc_auc_score"` is defined in binary classification for instance.